docs/user/library/coverage/image.rst

```diff
@@ -7,9 +7,8 @@ This is the quickest way to get a GIF or JEPG onto a map display - add a small t
 file along side the image defining the image's extend and use this plugin.
 
 **References**
-
```

Comment:
RST required line breaks to seperate sections such as headers and bullets, so I think this is going to cause a problem (you may want to build the docs locally to confirm); keep the line break.